app/client/views/confirmation/confirmation.html

```diff
@@ -327,15 +345,30 @@
               <p>
                 So that we can process your reimbursement faster, please name your receipt: <strong>{{fileName}}</strong>
               </p>
+              <p>
+                This doesn't need to be done immediately, you can come back and upload it at any time!
+              </p>
               <a
                 target="_blank"
-                href="https://go.hackmit.org/travel-receipts"
+                href="/travel-receipts"
```

Comment:
Once again, can we just link directly instead of providing routes?

Reply:
yup, sure can